build.gradle.kts

```diff
@@ -1,15 +1,14 @@
 plugins {
-    id("java-library")
-    id("maven-publish")
-    id("signing")
+    `java-library`
+    `maven-publish`
+    signing
```

Comment:
Why is this not unified anymore? :(

Reply:
sticking to the official conventions also used in the Gradle documentation